Core/Analyzers/CategoryAnalyzer.cpp

```diff
@@ -0,0 +1,81 @@
+#include <ObjectiveNinjaCore/Analyzers/ClassAnalyzer.h>
+#include <ObjectiveNinjaCore/Analyzers/CategoryAnalyzer.h>
+
+#include <ObjectiveNinjaCore/TypeParser.h>
```

Comment:
Also unnecessary to `#include` (although `ClassAnalyzer.cpp` makes this mistake too, not your fault)